package.json

```
@@ -74,7 +74,8 @@
       "path-to-regexp@>=0.1.7": "^0.1.12",
       "path-to-regexp@>=6.3.0": "^6.3.0",
       "next": "^15.2.3",
-      "form-data": "^4.0.4"
+      "form-data": "^4.0.4",
+      "sha.js@2.4.11": "2.4.12"
```

```suggestion
      "sha.js": ">=2.4.12"
```
so we always use a version above `2.4.12`